syntax = "proto3";

message GetUserRequest {
  int32 user_id = 1;
}

message AddUserRequest {
  string name = 1;
  string email = 2;
}

message UserData {
  string name = 1;
  string email = 2;
}

service YourService {
  rpc GetUser(GetUserRequest) returns (UserData);
  rpc AddUser(AddUserRequest) returns (UserData);
}